packages/webcomponents/src/components/order-terminals/order-terminals.tsx

```diff
@@ -63,27 +79,101 @@ export class OrderTerminals {
           message: error,
           severity,
         });
-        this.isLoading = false;
+        this.businessIsLoading = false;
       }
     });
   }
 
-  render() {
-    if (this.errorMessage) {
-      return <StyledHost>{ErrorState(this.errorMessage)}</StyledHost>;
-    }
+  private initializeGetTerminalModels() {
+    const getTerminalModels = makeGetTerminalModels({
+      id: this.businessId,
+      authToken: this.authToken,
+      service: new TerminalService(),
+    });
 
-    if (this.isLoading) {
-      return (
-        <StyledHost>
-          <p>Loading...</p>
-        </StyledHost>
-      );
-    }
+    getTerminalModels({
+      onSuccess: ({ terminals, orderLimit }) => {
+        this.terminalModels = terminals;
+        this.orderLimit = orderLimit;
+        this.terminalsIsLoading = false;
+      },
+      onError: ({ error, code, severity }) => {
+        this.errorMessage = error;
+
+        this.errorEvent.emit({
+          errorCode: code,
+          message: error,
+          severity,
+        });
+        this.terminalsIsLoading = false;
+      }
+    });
+  }
 
+  private onSelectedQuantityChange(event) {
+    this.order.updateTerminal(event.detail.terminalId, event.detail.quantity);
+    this.totalQuantity = this.order.totalQuantity;
+  }
+
+  render() {
     return (
       <StyledHost>
-        <p>Business Legal Name: {this.business.legal_name}</p>
+        {this.businessIsLoading && <OrderTerminalsLoading />}
+
+        {!this.businessIsLoading && this.errorMessage && ErrorState(this.errorMessage)}
+
+        {!this.businessIsLoading && !this.errorMessage && this.business && (
+          <div class="gap-5 pt-5" part={text}>
+            <div class="row">
+              <div class="col-12">
+                <h2 part={heading4}>Business Information:</h2>
+                <p>Business Name: {this.business.legal_name}</p>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-6">
+                <h2 part={heading4}>Representative:</h2>
+                <ul class="list-group" part={listGroup}>
+                  <li class="list-group-item" part={listGroupItem}>Name: {this.business?.representative?.name}</li>
+                  <li class="list-group-item" part={listGroupItem}>Title: {this.business?.representative?.title}</li>
+                  <li class="list-group-item" part={listGroupItem}>Email: {this.business?.representative?.email}</li>
+                  <li class="list-group-item" part={listGroupItem}>Phone: {this.business?.representative?.phone}</li>
+                </ul>
+              </div>
+              <div class="col-6">
+                <h2 part={heading4}>Shipping Address:</h2>
+                <ul class="list-group" part={listGroup}>
+                  <li class="list-group-item" part={listGroupItem}>Street: {this.business.legal_address?.line1 || ''}</li>
```

Comment:
Is the `|| ''` condition necessary? I think with optional chaining it'll either have a value or be `undefined` which should not show up on the page